build.sh

```diff
@@ -18,12 +18,12 @@ set -e
 
 # lint yaml of charts
 helm lint ./nxrm-aws-resiliency
-#helm lint ./nexus-repository-manager
+helm lint ./nexus-repository-manager
 
 # unit test
 #(cd ./nxrm-aws-resiliency; helm unittest -3 -t junit -o test-output.xml .)
 #(cd ./nexus-repository-manager; helm unittest -3 -t junit -o test-output.xml .)
```

Comment:
This line needs to be uncommented as well.

Reply:
There's a commented `testResults` line in the Jenkinsfile that should also be enabled. It will collect up these test output files for display in jenkins.